### DISCUSSION TITLE
code server can not run on android pad

### DISCUSSION THREAD

**Discussion by sinojelly** [coder/code-server#2616] [Q&A]
I use Huawei MatePad Android Pad, run Ubuntu 20.04 on Termux.
Then use yarn to install code server, see attached picture 1.
Run command : code-server --auth none
It says yazl, node-browser can not found, please see attached picture 2 & 3.
Any help would be appericiated!
Picture 1:
![Screenshot_20210121_071704_com termux](https://user-images.githubusercontent.com/520711/105367878-8badaf80-5bc6-11eb-8214-493190380763.jpg)
Picture 2:
![Screenshot_20210121_223351_com termux](https://user-images.githubusercontent.com/520711/105368022-b1d34f80-5bc6-11eb-86b8-ccde0040b9cd.jpg)
Picture 3:
![Screenshot_20210121_223411_com termux](https://user-images.githubusercontent.com/520711/105368047-bbf54e00-5bc6-11eb-844a-2bc2f2816f09.jpg)

**Comment by code-asher** [comment]
It looks like it failed to download the VS Code dependencies. We have a
bug where the install will keep going even if the dependencies fail to
install. Trying to install again might resolve the issue. You could also
try moving into the `lib/vscode` directory itself inside code-server
(you can find the location with `yarn global dir`) and run `yarn`.

**Reply by sinojelly** [reply]
Thanks for your quick reply.
I tried to run yarn in lib/vscode dir, it prompt an error like this: 
/usr/bin/node[18408]: ../src/node.cc:699:void node::ResetStdio(): Assertion((*__errno_location ())) == (9)' failed.
![Screenshot_20210122_063438_com termux](https://user-images.githubusercontent.com/520711/105421121-32b23b80-5c07-11eb-82c9-4940ab8a38da.jpg)

**Reply by code-asher** [reply]
That's an intense error. I'm really not sure how it can be fixed, unfortunately. I've had a lot of trouble building Node modules on Android. You could also try running `npm rebuild` but that's a shot in the dark and probably won't work either.
